### ISSUE TITLE
Scaling for NUFFT operator

### ISSUE THREAD

**Issue by chaithyagr** [mrphys/tensorflow-nufft#35]
I feel it will help a lot to have scaling coded up inside the nufft operator, similar to https://github.com/chaithyagr/tfkbnufft/blob/da8de17bc5cb738d11150662d0876bec9efb54d8/tfkbnufft/nufft/fft_functions.py#L159 and https://github.com/chaithyagr/tfkbnufft/blob/da8de17bc5cb738d11150662d0876bec9efb54d8/tfkbnufft/nufft/fft_functions.py#L197 as it will help a lot to make sure that Op and AdjOp can match the scales for `ortho` case.

**Comment by jmontalt** [comment]
Yes, I agree that a scaling option might be useful. This is easy to implement as a simple wrapper in Python, but in the long term we could incorporate it into the kernel and potentially get some extra performance.
